modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -356,6 +357,26 @@ public void testExportDLReferencesInvalidReference() throws Exception {
 			true, true);
 	}
 
+	@Test
+	public void testExportInvalidUUIDDLReference() throws Exception {
+		_portletDataContextExport.setZipWriter(new TestReaderWriter());
+
+		String content = _replaceParameters(
+			_getContent("dl_references.txt"), _fileEntry);
+
+		try {
+			_exportImportContentProcessor.validateContentReferences(
+				_stagingGroup.getGroupId(), content);
+		}
+		catch (Exception exception) {
+			Assert.assertFalse(
+				"Filename should not be recognised as UUID",
+				exception.getCause() instanceof NoSuchFileEntryException);
+		}
+
+		Assert.assertTrue("Filename should not be recognised as UUID", true);
+	}
```

Comment:
Sorry I didn't explain myself correctly. My problem with this code is that you're checking that the code is throwing anything but a `NoSuchFileEntryException`. This will hide any unexpected error.

Can you please change the test so that the code checks the expected exceptions explicitly? Use the `expected` attribute in the `@Test` annotation for that. Also, remove the dummy `assertTrue` check.

Reply:
Hey @adolfopa ,

My goal was to catch the _NoSuchFileEntryException_ and warn the dev about the issue, I don't actually expect an exception to happen, only when the test fail. If i use _@Test(expected = NoSuchFileEntryException.class)_, the test will fail if I don't receive the _NoSuchFileEntryException_.

After I changed the random file name to the uuid pattern in the test file, our _testExportDLReferences_ test fails without the fix, however it only tells the dev that the test failed due to a _NoSuchFileEntryException_. I wanted to make it more verbose, but now I am hesitant wether It adds any value to the tests.
If we get an exception from __exportImportContentProcessor.validateContentReferences_(), a lot of test will fail already anyway.

Reply:
@NemethNorbert Tests should check specific conditions under controlled contexts. Can you write the test so that it sets up an invalid state where you expect the exception will be thrown, and check that it is thrown explicitly? If necessary, you can write a test setting up a valid context and then checking that it doesn't throw the exception.

I find you usage of assertions very confusing.

Reply:
Hey @adolfopa ,

I added a new test, where I create a fileEntry with a filename that matches the UUID pattern. This test will fail without the fix that was commited under LPS-186154 and it passes with it.

Please review it. Thanks